local-rest-scorer/src/test/java/ai/h2o/mojos/deploy/local/rest/controller/ModelsApiControllerTest.java

```diff
@@ -0,0 +1,174 @@
+package ai.h2o.mojos.deploy.local.rest.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import ai.h2o.mojos.deploy.common.rest.model.CapabilityType;
+import ai.h2o.mojos.deploy.common.transform.MojoScorer;
+import ai.h2o.mojos.deploy.common.transform.SampleRequestBuilder;
+import ai.h2o.mojos.deploy.common.transform.ShapleyLoadOption;
+import ai.h2o.mojos.runtime.MojoPipeline;
+import ai.h2o.mojos.runtime.api.BasePipelineListener;
+import ai.h2o.mojos.runtime.api.MojoPipelineService;
+import ai.h2o.mojos.runtime.frame.MojoColumn;
+import ai.h2o.mojos.runtime.frame.MojoFrame;
+import ai.h2o.mojos.runtime.frame.MojoFrameBuilder;
+import ai.h2o.mojos.runtime.frame.MojoFrameMeta;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.ResponseEntity;
+
+@ExtendWith(MockitoExtension.class)
+class ModelsApiControllerTest {
+  private static final String MOJO_PIPELINE_PATH = "src/test/resources/multinomial-pipeline.mojo";
+  private static final String TEST_UUID = "TEST_UUID";
+
+  @Mock private SampleRequestBuilder sampleRequestBuilder;
+
+  @BeforeAll
+  static void setup() {
+    System.setProperty("mojo.path", "src/test/resources/multinomial-pipeline.mojo");
+    mockDummyPipeline();
```

Comment:
Doesn't look like the static mock session was closed.  Not a big deal since the class is small and each test relies on this.

But for thoroughness, I'll share a couple of ways through this:

1. Perform staticmock as part of a `try()` at the place it's needed, which auto cleans up the mock after use.  [Example.](https://github.com/h2oai/h2oai-serving/blob/2f9c0b8590ff5800cc10718769d4f023bdcf958a/common/src/test/java/ai/h2o/deploy/common/environment/EnvironmentsTest.java#L55)
2. Save the mock to a static variable and close the session as part of a `@AfterAll`.